data/hai/hai reveal 5 philosophers.txt

```diff
@@ -509,9 +507,9 @@ mission "Hai Rescue [B05] Plant a Spy"
 	on offer
 		conversation
 			`You reach the spaceport with Teeneep and head to the room you were told to visit. A Hai stands before you, nervously eyeing the strange orb you're holding. "You must be Tutitot," Teeneep says. The Hai nods. "This is Captain <first> <last>. While you two work together, I will stay behind to prepare for our next steps."`
-			`	Tutitot addresses you. "The Quarg told me I'm supposed to put the worm in my ear. It's just so... weird and gross." You give him the orb, and he lifts it up and brings it near his head. The worm immediately jumps out and vanishes into his ear. The Hai stiffens for a few seconds, and then returns to normal. "I can... feel your mind. I can see the orb in my mind. It's like the whole world is in my mind! This is amazing!`
-			`	"I'm ready to go. Please deliver the orb to <stopovers> and then take me to <destination>. When I'm at the Philosopher's Paradise, the orb will... touch your mind? It will probably be two or three weeks after you place the it. Then you will return to Devil-Eye to contact me."`
-			`	Tutitot walks toward your ship as if in a trance. Teeneep follows from a distance, watching the Hai's odd behavior. After a moment, she turns to you. "You will be paid well upon your return," says Teeneep. "Please hurry."`
+			`	Tutitot addresses you. "The Quarg told me I'm supposed to put the worm in my ear. It's just so... weird and gross." You consult the instruction card and only now realize exactly what that step was. The pair of you look at that step and share a look before you give him the orb. Cautiously he lifts it up and brings it near his head. The worm immediately jumps out and vanishes into his ear. The Hai stiffens for a few seconds, and then returns to normal. "I can... feel the orb in my mind. It's as if it is expectant, waiting, but it seems to see the world around it; in fact it's like the whole world is in my mind! This is amazing!"`
```

Comment:
```suggestion
			`	Tutitot addresses you. "The Quarg told me I'm supposed to put the worm in my ear. It's just so... weird and gross." You consult the instruction card and only now realize exactly what that step was. The pair of you look at that step and share a look before you give him the orb. Cautiously, he lifts it up and brings it near his head. The worm immediately jumps out and vanishes into his ear. The Hai stiffens for a few seconds, and then returns to normal. "I can... feel the orb in my mind. It's as if it's expectant, waiting, but it seems to see the world around it; in fact, it's like the whole world is in my mind! This is amazing!"`
```

Reply:
I find that in speech people typically don't say "it's" twice so close to each other and will avoid the contraction on the second use, especially when trying to explain something. The exception being the particular phrasing of `"It's like it's"`, so I'd rather either keep it separate or switch over entirely.

Reply:
Fair point. I personally prefer `It's like it's` over keeping as-is, but up to you:
```suggestion
			`	Tutitot addresses you. "The Quarg told me I'm supposed to put the worm in my ear. It's just so... weird and gross." You consult the instruction card and only now realize exactly what that step was. The pair of you look at that step and share a look before you give him the orb. Cautiously he lifts it up and brings it near his head. The worm immediately jumps out and vanishes into his ear. The Hai stiffens for a few seconds, and then returns to normal. "I can... feel the orb in my mind. It's like it's expectant, waiting, but it seems to see the world around it; in fact, it's like the whole world is in my mind! This is amazing!"`
```
```suggestion
			`	Tutitot addresses you. "The Quarg told me I'm supposed to put the worm in my ear. It's just so... weird and gross." You consult the instruction card and only now realize exactly what that step was. The pair of you look at that step and share a look before you give him the orb. Cautiously he lifts it up and brings it near his head. The worm immediately jumps out and vanishes into his ear. The Hai stiffens for a few seconds, and then returns to normal. "I can... feel the orb in my mind. It's as if it is expectant, waiting, but it seems to see the world around it; in fact, it's like the whole world is in my mind! This is amazing!"`
```